recipe/0002-outcome-account-for-void-types.patch

```diff
@@ -0,0 +1,187 @@
+diff --git boost_1_79_0/boost/outcome/detail/revision.hpp boost_1_79_0/boost/outcome/detail/revision.hpp
+index 1664273d..c6738b9e 100644
+--- boost_1_79_0/boost/outcome/detail/revision.hpp
++++ boost_1_79_0/boost/outcome/detail/revision.hpp
+@@ -22,6 +22,6 @@ Distributed under the Boost Software License, Version 1.0.
+ */
+ 
+ // Note the second line of this file must ALWAYS be the git SHA, third line ALWAYS the git SHA update time
+-#define OUTCOME_PREVIOUS_COMMIT_REF 147ec1e8673c34cb7cf431dfdbf211d8072d7656
+-#define OUTCOME_PREVIOUS_COMMIT_DATE "2022-03-17 19:38:05 +00:00"
+-#define OUTCOME_PREVIOUS_COMMIT_UNIQUE 147ec1e8
++#define OUTCOME_PREVIOUS_COMMIT_REF 5e36246352c1c412813583e446eea0d1a19d08cc
++#define OUTCOME_PREVIOUS_COMMIT_DATE "2022-04-05 06:23:15 +00:00"
++#define OUTCOME_PREVIOUS_COMMIT_UNIQUE 5e362463
```

Comment:
drop the revision changes

Reply:
Revision changes are correct but the macro was renamed `BOOST_OUTCOME_ccc` in the meantime.

Reply:
check the git repo